### PR TITLE
select: Fix a11y markup

### PR DESCRIPTION
**Pull request by aristov**

### CODE REVIEW

**Comment by aristov** [comment]
#1734

**Comment by tadatuta** [comment]
There's codestyle error in PR

**Comment by aristov** [comment]
> There's codestyle error in PR

Where?

**Comment by tadatuta** [comment]
https://travis-ci.org/bem/bem-components/jobs/104734867#L592

File: common.blocks/select/select.bemhtml [hunk 1]
```diff
@@ -21,7 +21,7 @@ block('select')(
                         iterateOptions(option.group);
                     } else {
                         firstOption || (firstOption = option);
-                        optionIds.push(option.id = _this.generateId());
+                        optionIds.push(option.id = 'option' + random++);
```
Comment (veged) [inline common.blocks/select/select.bemhtml]: instead of `'option' + random++` (with `random` variable) we can just use `_this.generateId() + 'option' + i`

Comment (aristov) [inline common.blocks/select/select.bemhtml]: If we have `group`, the `i` iterator will starts from zero. `id` attribute will not be unique in this case.

Comment (veged) [inline common.blocks/select/select.bemhtml]: yeah... so even better, you can use `option.id = _this.identify(option)`!

Comment (aristov) [inline common.blocks/select/select.bemhtml]: What about `bh`? There is no `identify` method.

Comment (veged) [inline common.blocks/select/select.bemhtml]: as I can see in [code](https://github.com/bem/bh/blob/dff393fa5c6c7f1d0a327a1c2f43895bcd23a0c7/lib/bh.js#L256) and [specs](https://github.com/bem/bh/blob/dff393fa5c6c7f1d0a327a1c2f43895bcd23a0c7/test/test.generateId.js#L10) `generateId` in BH works different (also incorrect) and generate different ids every time — so we can just do `option.id = ctx.generateId()`

Comment (aristov) [inline common.blocks/select/select.bemhtml]: Done.

**Comment by veged** [comment]
now changes looks good for me —@aristov, can you rebase and merge?

**Comment by narqo** [comment]
@aristov for future works, please, squash your commits, before merging, so PR'd be more solid.

**Pull request merged**